activities/staff-meetings/event-debrief.md

```diff
@@ -0,0 +1,25 @@
+---
+type: Resource
+explains: This is a template agenda for a staff debrief after an event
+---
+
+# Template agenda
+
+1) New contacts
+* who are the interesting people we met?
+* why do we think they are interesting for us as an organization?
+
+2) What could the event and our participation in it mean for
+* membership
+* codebase stewardship
+* the Standard for Public Code
+* developing our community
+* communications
+* improving About
+
+3) Next steps
+* generate overview of identified next steps 
+* assign next steps to a team of owners (at least 2 staff members)
+* define a timeframe, if relevant
+
+4) Decide if it would be useful to run a retro for this event
```

Comment:
This is a bit of a difficult thing in markdown but will parse differently in different engines and for that reason is not recommended. It is ambiguous as to whether this should be interpreted as an ordered list with sub bullets or multiple lists with headings.

I would recommend one of the following formatting options:

```markdown
1) A
  * b
  * c
2) D
  * e
```

Or alternatively:

```
## 1) A

* b
* c

## 2) D

* e
```

Additionally, headers or lists should always have whitespace around them.

I understand this is a bit of a hassle, however this is exactly what makes markdown portable, it forces you to use a limited set of structural elements that can then be designed in a system.

I can explain more if you want to.

Reply:
Thanks for the explanation! fixed in a different pull request